src/config/AppConfiguration.js

```diff
@@ -23,9 +25,15 @@ export const ENABLE_APPLICATION_INSIGHTS = true;
 // Support page url
 export const SUPPORT_URL = 'http://support.cosmotech.com';
 
+// Support page url
+export const COSMOTECH_URL = 'http://cosmotech.com';
+
 // Documentation url
 export const DOCUMENTATION_URL = 'doc.pdf';
 
+// Is there "About" section
+export const ABOUT = About;
+
```

Comment:
Replacing the default About component by a custom About component is trickier than a simple text configuration, it requires some React development.
We should keep this configurable but not in the file `src/config/AppConfiguration.js` (this file should remain string values only, not import if we can avoid it).

Could you create a new file in `src/services/config` (e.g. Menu.js) and put the component import and export there ?